code/modules/gear_presets/other.dm

```diff
@@ -845,38 +845,6 @@
 
 //*****************************************************************************************************/
 
-/datum/equipment_preset/other/professor_dummy
```

Comment:
Removing this means it is no longer in the create humans menu; which is generally how its been spawned. It'd probably be better to just keep the preset but have your closest spawn the equipment preset into the closet.

Reply:
See https://github.com/cmss13-devs/cmss13/blob/master/code/modules/admin/topic/topic_events.dm#L151-L159 how create_humans uses these presets. You likely just need to create the mob, and then `arm_equipment` it.

Reply:
If the concern was just not registering the signal for admin spawned; then have the closet register the signal for that spawn on the dummy's behalf (as in we aren't registering a signal on the closet, but the closet's initialize call is having the dummy register the signal)

Reply:
Relayed on discord, but putting it here too. Something like
```js
/obj/structure/closet/secure_closet/professor_dummy/Initialize()
    . = ..()
    var/mob/living/carbon/human/dummy/professor_dummy/spawned_dummy = new(src)
    var/datum/equipment_preset/dummy_preset = /datum/equipment_preset/other/professor_dummy
    arm_equipment(spawned_dummy, initial(dummy_preset.name), TRUE, FALSE)
    spawned_dummy.RegisterSignal(SSdcs, COMSIG_GLOB_HIJACK_LANDED, TYPE_PROC_REF(/mob/living/carbon/human/dummy/professor_dummy, destroy_upon_hijack))```
should work. The signal registration here is just if we want only closet dummies to dust; but I don't mind admin spawns having the same signal either if they're on a game level z.